sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/data-load.sh

```diff
@@ -84,3 +86,7 @@ echo "-----------------------------"
 ./bin/ycsb load cassandra-cql -p hosts=${external_ip} -p dataintegrity=true -p recordcount=\
 ${recordcount} -p insertorder=ordered -p fieldlength=20 -P workloads/workloadd \
 -s > workloada_load_res.txt
+
+# Delete cassandra sinle node set up
+cd ../LargeITCluster
+kubectl delete -f cassandra-svc-temp.yaml
```

Comment:
One solution is that it may be appropriate for this script to not fail on the first error